waspc/data/Generator/templates/server/src/auth/providers/email/login.ts

```diff
@@ -36,6 +36,8 @@ export function getLoginRoute() {
     
         const auth = await findAuthWithUserBy({ id: authIdentity.authId })
         const session = await createSession(auth.id)
+
+        // postLogin(providerId)
```

Comment:
After we create the session, execute `postLogin`

Reply:
The param `providerId` is something like `{ providerName: "username", providerUserId: "miho" }`